data/json/items/armor.json

```diff
@@ -7170,6 +7192,33 @@
     "material_thickness": 1,
     "flags": [ "OVERSIZE", "BELTED", "RESTRICT_HANDS", "WATER_FRIENDLY" ]
   },
+  {
+    "id": "police_belt",
+    "type": "ARMOR",
+    "name": "police duty belt",
+    "description": "Black leather belt used by police officers.  It has several pouches and a holder for a baton.",
+	"looks_like": "fireman_belt",
+    "weight": 250,
+    "volume": 8,
+    "price": 10000,
+    "bashing": 5,
+    "material": [ "leather", "steel" ],
+    "symbol": "[",
+    "color": "brown",
+    "covers": [ "TORSO" ],
+    "encumbrance": 4,
+    "storage": 8,
+    "use_action": {
+      "type": "holster",
+      "holster_prompt": "Attach what to holder?",
+      "holster_msg": "You attach your %s to your %s",
```

Comment:
Period in the end.

Reply:
Corrected. It is a copy-paste from fireman_belt so it would need change there also.